### PR TITLE
ACMS-1705: The icons are missing when site is installed from the existing configurations.

### PR DESCRIPTION
**Pull request by chandan-singh7929**
**Motivation**

Fixes #[ACMS-1705](https://backlog.acquia.com/browse/ACMS-1705)

**Proposed changes**

This change help export drupal standard config and site studio configuration in separate directory and its provides drush post command that import/export site studio package on config import/export.

**Alternatives considered**
Manually do the configuration and run drush command separelty.

**Testing steps**
export your site studio keys
./vendor/bin/acms site:install --yes
./vendor/bin/drush en acquia_cms_site_studio_cm
./vendor/bin/drush cget system.site  and copy uuid
./vendor/bin/drush cex --yes
./vendor/bin/drush site:install minimal --yes
./vendor/bin/drush cset system.site uuid <uuid>--yes
./vendor/bin/drush cim --yes

**Merge requirements**
- [ ] _Major change_, _Minor change_, _Bug_, _Enhancement_, and/or _Chore_ label applied
- [ ] Manual testing by a reviewer

### CODE REVIEW

File: tests/ci/script.sh [hunk 1]
```diff
@@ -23,7 +23,7 @@ fi
 [[ -d "${ORCA_FIXTURE_DIR}" ]] || exit 0
 
 cd ${ORCA_FIXTURE_DIR}
-if [ "${ACMS_JOB}" == "backstop_tests" ] || [ "${ACMS_JOB}" == "upgrade_modules" ]; then
+if [ "${ACMS_JOB}" == "backstop_tests" ] || [ "${ACMS_JOB}" == "upgrade_modules" ] || [ "${ACMS_JOB}" != "blt" ]; then
```
Comment (rajeshreeputra) [inline tests/ci/script.sh:26]: please change to 
`if [ "${ACMS_JOB}" == "backstop_tests" ] || [ "${ACMS_JOB}" == "upgrade_modules" ] || [ "${ACMS_JOB}" == "blt" ]; then`

**Review by rajeshreeputra — APPROVED** [review]
Changes looks good to me

**Review by deepakmishra2 — APPROVED** [review]
changes looks good

File: modules/acquia_cms_site_studio/composer.json [hunk 2]
```diff
@@ -53,7 +50,8 @@
         "enable-patching": true,
         "patches": {
             "acquia/cohesion": {
-                "3340269 - Site Studio: Allow optional packages to import": "https://www.drupal.org/files/issues/2023-03-16/3340269-site-studio-allow-optional-packages-6.patch"
+                "3340269 - Site Studio: Allow optional packages to import": "https://www.drupal.org/files/issues/2023-03-16/3340269-site-studio-allow-optional-packages-6.patch",
+                "2571235 - Roles should depend on objects that are building the granted permissions": "https://gist.githubusercontent.com/vishalkhode1/e1ac95e3cbac97a06cd52430e3018b57/raw/af1f374c5263fc67f12b1f36b22144b57bc8714a/site-studio-dynamic-permissions.patch"
```
Comment (vishalkhode1) [inline modules/acquia_cms_site_studio/composer.json:54]: @chandan-singh7929 We shouldn't add gist permissions in acquia_cms module composer.json. Please add in root composer.json.

File: modules/acquia_cms_common/src/EventSubscriber/ConfigEventsSubscriber.php [hunk 3]
```diff
@@ -197,6 +197,13 @@ public function configSave(ConfigCrudEvent $event) {
         }
       }
     }
+    // Clear cache once acquia purge.
+    else {
+      $config = $event->getConfig();
+      if ($config->getName() == 'purge.plugins') {
```
Comment (vishalkhode1) [inline modules/acquia_cms_common/src/EventSubscriber/ConfigEventsSubscriber.php:203]: @chandan-singh7929 Add todo here. I don't like the idea of clearing cache in event subscriber. That's not good approach.
cc: @rajeshreeputra

File: modules/acquia_cms_common/acquia_cms_common.module [hunk 4]
```diff
@@ -490,20 +422,12 @@ function acquia_cms_common_module_implements_alter(array &$implementations, stri
...
-
-/**
- * Implements hook_css_alter().
- */
-function acquia_cms_common_css_alter(array &$css) {
-  if (\Drupal::service('module_handler')->moduleExists('collapsiblock')) {
-    // Remove the default css provided by module: `collapsiblock`.
-    $path = \Drupal::service('extension.list.module')->getPath('collapsiblock') . '/theme/dist/css/collapsiblock.min.css';
-    unset($css[$path]);
+  // during site installation & configuration synchronization
```
Comment (vishalkhode1) [inline modules/acquia_cms_common/acquia_cms_common.module:425]: @chandan-singh7929 Add todo here with ACO ticket number mentioning that we need to revert once that ticket is released.

**Review by vishalkhode1 — CHANGES_REQUESTED** [review]

File: modules/acquia_cms_common/src/EventSubscriber/ConfigEventsSubscriber.php (hunk 3 above)
Comment (chandan-singh7929) [reply modules/acquia_cms_common/src/EventSubscriber/ConfigEventsSubscriber.php:205]: Done

File: modules/acquia_cms_site_studio/composer.json (hunk 2 above)
Comment (chandan-singh7929) [reply modules/acquia_cms_site_studio/composer.json]: Done

File: modules/acquia_cms_common/acquia_cms_common.module (hunk 4 above)
Comment (chandan-singh7929) [reply modules/acquia_cms_common/acquia_cms_common.module:425]: Done

**Review by vishalkhode1 — APPROVED** [review]

**Pull request merged**